packages/gatsby-theme-carbon/src/components/SquareCard/square-card.scss

```
@@ -0,0 +1,185 @@
+// Square Card Group
+.square-card-group .#{$prefix}--col-lg-4 {
```

```suggestion
.square-card-group > *
```

Better to be less specific in case the number of columns changes. We're really just trying to target all the direct children, we don't super care the number of columns.